2/index.js

```diff
@@ -0,0 +1,14 @@
+console.log("No.2");
+// Main-Program
+// node作成
+const list = document.querySelector("main > ul");
+const listChild_node = document.createElement("li");
+const alink_node = document.createElement("a");
+const img_node = document.createElement("img");
+
+listChild_node.textContent = "これです";
+alink_node.setAttribute("href", "1.html");
```

Comment:
nits
I believe the camel case is more common in JavaScript.

Reply:
I got it! Thanks!